ucte/ucte-converter/src/test/java/com/powsybl/ucte/converter/UcteExporterTest.java

```diff
@@ -35,30 +35,31 @@
     private static Network iidmNetwork;
     private static Network iidmTieLineNetwork;
     private static Network iidmSwitchNetwork;
+    private static Network mergedNetwork;
     private UcteExporter ucteExporter = new UcteExporter();
 
     @Rule
     public final ExpectedException exception = ExpectedException.none();
 
     @BeforeClass
     public static void setUpBeforeClass() {
-        ReadOnlyDataSource dataSource = new ResourceDataSource("transformerRegulation", new ResourceSet("/", "transformerRegulation.uct"));
-        transfomerRegulationNetwork = new UcteImporter().importData(dataSource, null);
-        dataSource = new ResourceDataSource("exportTest", new ResourceSet("/", "exportTest.uct"));
-        exportTestNetwork = new UcteImporter().importData(dataSource, null);
+        transfomerRegulationNetwork = Importers.loadNetwork("transformerRegulation.uct", UcteExporterTest.class.getResourceAsStream("/transformerRegulation.uct"));
```

Comment:
Why do you have replaced UcteImporter + ResourceDataSource by Importers.loadNetwork?

Reply:
Good question, it seemed simpler, and as this class is not dealing with UcteImporter test, but exporter one I thought it was more readable, but I can still change that if you prefer the old method.

Reply:
Beware, I think it's one of those naughty methods which innocently loads some config behind your back ;) (`see ImportConfig`).

Reply:
Importers.loadNetwork is ok but you should use an overload because this one could load data from default platform config

Reply:
> Importers.loadNetwork is ok but you should use an overload because this one could load data from default platform config

And also, a difference between the approaches is that loadNetwork make an in memory copy of file but using ResourceDataSource data is streamed without any copy. But in case of small test case it is not really relevant.

Reply:
Yeah, but to be honnest it will be a real pain in the a.. to use these naughty data sources that are not really good either (cf the discussions we have to totally rethink the data sources ...) in all the tests for importing network data.
I am not really convinced by the solution proposed, we should provide an implementation that does not depend on the configuration...

Reply:
I rolled back to the previous version of the import method for this PR, but it should be nice to get the equivalent of the loadNetwork wrapper without the constraints of configuration dependency. Any IDEA on possible solution ?

Reply:
You can use this one:
```java
public static Network loadNetwork(String filename, InputStream data, ComputationManager computationManager, ImportConfig config, Properties parameters, ImportersLoader loader);
 ```

Reply:
You can refer to `TerminalRefTest.java` to see how to use `Importers.loadNetwork()` independently to the local configuration. Though it is easier to use `new UcteImporter().importData`